### PR TITLE
[Sample app Kotlin] separate sample-kotlin module

### PR DESCRIPTION
**Pull request by lukaszkalnik**
Creates a separate `sample-kotlin` module. As a start, `ScanActivity` and `DeviceActivity` are implemented, with parts of functionalities outcommented to make for an easier review.

Also all layouts have been copied over from the Java sample and cleaned up a little bit.

### CODE REVIEW

**Comment by dariuszseweryn** [comment]
I have added `Work In Progress` to the title as I have seen that there are some `TODO`s in the code. Feel free to rename when you finish
Have a Happy New Year! :)

**Comment by dariuszseweryn** [comment]
In this case I will put it on top of my Open Source priority list

File: sample-kotlin/src/main/java/com/polidea/rxandroidble2/samplekotlin/example1_scanning/ScanActivity.kt [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    private fun configureResultList() {
+        with(recyclerView) {
+            setHasFixedSize(true)
+            itemAnimator = null
+            layoutManager = LinearLayoutManager(this@ScanActivity)
+            adapter = resultsAdapter
+        }
+    }
+
+    private fun onAdapterItemClick(scanResult: ScanResult) {
```
Comment (dariuszseweryn) [inline sample-kotlin/src/main/java/com/polidea/rxandroidble2/samplekotlin/example1_scanning/ScanActivity.kt]: This seems not used?

Comment (lukaszkalnik) [reply sample-kotlin/src/main/java/com/polidea/rxandroidble2/samplekotlin/example1_scanning/ScanActivity.kt]: Just forgotten to remove it.

**Comment by dariuszseweryn** [comment]
One last thing — I have just released version 1.8.0 which is now using `androidx`. I have migrated the java sample app already. Could you also update the kotlin one? From now on I do not expect more such changes

**Pull request merged**